zerver/webhooks/zendesk/doc.md

```diff
@@ -1,51 +1,50 @@
+# Zulip Zendesk integration
+
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. Construct a webhook URL like the following:
-
-    `{{ api_url }}/v1/external/zendesk?ticket_title={% raw %}{{ ticket.title }}&ticket_id={{ ticket.id }}{% endraw %}`
+1. {!generate-webhook-url-basic.md!}
 
 1. {!append-channel-name.md!}
 
-1. Next, in Zendesk, open your **Admin** view via gear in the bottom-left
-    corner. In the **Admin** view, click on **Extensions**, then click
-    **add target**.
+1. In Zendesk, click the **gear** (<i class="fa fa-cog"></i>) icon in the
+    bottom-left corner, click on **Extensions**, and then click **add target**.
 
     ![](/static/images/integrations/zendesk/001.png)
     ![](/static/images/integrations/zendesk/002.png)
 
-1. From there, click **URL target**. Fill in the form like this:
+1. Click the **URL target**. Fill in the form like this:
 
     * **Title**: Zulip
-    * **URL**: the URL we created above
+    * **URL**: the URL generated above
     * **Method**: POST
     * **Attribute Name**: message
     * **Username**: your bot's user name, e.g., `zendesk-bot@yourdomain.com`
     * **Password**: your bot's API key
 
     ![](/static/images/integrations/zendesk/003.png)
 
-1. Now, select **Test Target** and click **Submit**. A test message should
-    appear in the `zendesk` channel. If the message was received, save the
```

Comment:
Removed the "If the message was received" condition, as we're not addressing the failure anyway. So, just mentioning that it's expected should be enough, I assume.